spec/predictor_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+describe Crystalball::Predictor do
+  subject { described_class.new(map, diff) }
+  let(:repository) { Git::Base.new }
+  let(:file_diff1) { Crystalball::SourceDiff::FileDiff.new(repository, Git::Diff::DiffFile.new(repository, path: 'file1.rb')) }
+  let(:diff) { [file_diff1] }
+  let(:map) { {spec_file: %w[file1.rb]} }
+
+  describe '#cases' do
+    subject { super().cases }
```

Comment:
that `super()` notation is a bit weird for me because you change the subject completely. I'd rather use `subject { predictor.cases }` in this case